### PR TITLE
Update Model.js to fix missing oldValue and add property name to call…

### PR DESCRIPTION
**Pull request by erotavlas**
…pback function

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (dylans) [inline .vscode/settings.json]: Please amend your PR to exclude this file.

File: Model.js [hunk 2]
```diff
@@ -453,9 +453,10 @@ define([
 					reactive.value = listener(this.valueOf());
 				}
 			}
-			// add to the listeners
-			var handle = this._addListener(function (value) {
-				var result = listener(value);
+		    // add to the listeners
```
Comment (dylans) [inline Model.js]: mix of tabs and spaces on this and the next line is impacting the formatting

File: Model.js [hunk 3]
```diff
@@ -533,7 +534,7 @@ define([
 				// notify listeners
 				if (property.onchange) {
 					// queue the callback
-					property._queueChange(property.onchange, oldValue);
+				    property._queueChange(property.onchange, oldValue, property.name);
```
Comment (dylans) [inline Model.js]: tabs not spaces

**Comment by erotavlas** [comment]
Hi Dylan, Please let me know if the changes were applied correctly
Thanks

**Comment by dylans** [comment]
> Hi Dylan, Please let me know if the changes were applied correctly
> Thanks

Hi @erotavlas, I'm not seeing a second commit to address my feedback...

**Comment by erotavlas** [comment]
Sorry , forgot to do a git push, the second commit should be there now.

**Pull request merged**